src/external/views/nunjucks/kpi-reporting/dashboard.njk

```diff
@@ -0,0 +1,223 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ title(pageTitle, caption) }}
+  </div>
+</div>
```

Comment:
Indentation could be more consistent to aid readability.

Reply:
Done - thanks